src/pps/pps.proto

```diff
@@ -776,9 +776,8 @@ message ListPipelineRequest {
   //-1: Return all historical versions.
   int64 history = 2;
 
-  // When true, return PipelineInfos with the details field, which requires
-  // loading the pipeline spec from PFS.
-  bool details = 3;
```

Comment:
This is technically a breaking change for the python client. If possible, I think it would be nicer to set the `[deprecated = true]` option on this field ([link](https://protobuf.dev/programming-guides/proto3/#options)). That way user's code won't break and they'll be presented with a `DeprecationWarning: ListPipelineRequest.details is deprecated` if the field is set.

Reply:
Good idea — I’ve switched to deprecation.